bookwyrm/templates/edit_user.html

```diff
@@ -42,6 +42,13 @@ <h1 class="title">Profile</h1>
                     {{ form.manually_approves_followers }}
                 </label>
             </p>
+            <p class="block">
+                <label class="label" for="id_default_post_privacy">
+                    Default post privacy:
+                    {% include 'snippets/privacy_select.html' with no_label=False current=user.default_post_privacy %}
```

Comment:
Considered actually making a change to `privacy_select.html` itself, but think this is a reasonable way to handle this. Open to feedback.

Reply:
I think it would make sense to edit the `privacy_select.html` snippet so that it could have `name="default_post_privacy"` here and `name="privacy"` when it's used in the create status snippet, that way you wouldn't have to change `bookwyrm/view_actions.py` at all (I refactored it a bit in https://github.com/mouse-reeve/bookwyrm/pull/418/commits/e6105c6cb0000a49c8aeb76411f57124a80947c1 to use django's form.save() functionality, rather than setting the fields manually, which I didn't realize was a thing when I started coding this 😂 )